### ISSUE TITLE
WCAG violation

### ISSUE THREAD

**Issue by bruffridge** [uswds/uswds#2334]
## Description

Line where issue occurred:
https://github.com/uswds/uswds/blob/develop/src/stylesheets/core/_base.scss#L15

Description of issue found via SortSite:
https://www.powermapper.com/products/sortsite/rules/acchtmlunsettextorbackgroundcolor/

More information on issue from https://www.w3.org/TR/WCAG20-TECHS/F24.html
Failure Example 1: Specifying only background color with CSS
In the example below the background color is defined on the CSS stylesheet, however the foreground color is not defined. Therefore, the example fails the Success Criterion.

Example Code:

  ```html
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" 
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<html>
<head>
 <title>Setting the canvas background</title>
    <style type="text/css">

       body {background-color:white}
    </style>
  </head>
  <body>
    <p>My background is white.</p>
  </body>
</html>
```

Proposed solution:
Add ```color: $color-black``` to body

**Comment by maya** [comment]
Hi @bruffridge, thanks for opening this issue. Our base font color is defined as `$color-base: #212121` in https://designsystem.digital.gov/components/colors/#palette so I'm also surprised it seems to be missing from the code base / not implemented. Thanks for pointing this out and we'll try to get this fixed.
